db/schema.rb

```diff
@@ -41,7 +41,7 @@
     t.datetime "updated_at", null: false
     t.index ["hypervisor"], name: "index_hw_infos_on_hypervisor"
     t.index ["system_id"], name: "index_hw_infos_on_system_id", unique: true
-    t.index ["uuid"], name: "index_hw_infos_on_uuid", unique: true
+    t.index ["uuid"], name: "index_hw_infos_on_uuid"
```

Comment:
I am thinking of doing this in a separate migration.. :thinking:

Reply:
OK, drop the index in a separate migration then.

Reply:
should be better like that. done